api/rosetta/errors.go

```diff
@@ -31,15 +31,18 @@ const (
 	addressLength    = "account identifier has invalid address field length"
 	currenciesEmpty  = "currency identifier list is empty"
 	symbolEmpty      = "currency identifier has empty symbol field"
-	txEmpty          = "transaction identifier has empty hash field"
+	txHashEmpty      = "transaction identifier has empty hash field"
 	txLength         = "transaction identifier has invalid hash filed length"
 	txInvalidOpCount = "transaction operations list has an invalid number of operations"
+	txEmpty          = "transaction text is empty"
+	txInvalid        = "transaction text is invalid"
```

Comment:
I don't like how you mix up transaction identifier and signed transaction here. Maybe you can use `txBodyEmpty` or `txBodyInvalid`. At least the naming should make a clear difference.